templates/zerver/help/import-from-gitter.md

```diff
@@ -1,117 +1,196 @@
-the Zulip documentation on
-[making a user an organization owner from the terminal][grant-admin-access]
-to mark the appropriate users as organization owners.
+    ```
+    cd /home/zulip/deployments/current
+    ./scripts/stop-server
+    ./manage.py convert_gitter_data /tmp/gitter_data.json --output /tmp/converted_gitter_data
+    ./manage.py import <subdomain> /tmp/converted_gitter_data
+    ./scripts/start-server
+    ```
 
-[grant-admin-access]: https://zulip.readthedocs.io/en/latest/production/management-commands.html#grant-administrator-access)
-[gitter-api-user-data]: https://developer.gitter.im/docs/user-resource
+1. Follow [step 4](https://zulip.readthedocs.io/en/stable/production/install.html#step-4-configure-and-use)
+   of the guide for [installing a new Zulip
+   server](https://zulip.readthedocs.io/en/stable/production/install.html).
+
+{end_tabs}
 
-## Caveats
+#### Import details
 
-- The [Gitter data export tool](https://github.com/minrk/archive-gitter)
-  doesn't support exporting private Gitter channels.
+Whether you are using Zulip Cloud or self-hosting Zulip, here are a few notes to
+keep in mind about the import process:
 
-- Gitter's export tool doesn't export email addresses; just GitHub
-  usernames.  The import tool will thus use [GitHub's generated
-  noreply email addresses][github-noreply] to compute the
-  GitHub-generated noreply email address associated with that GitHub
-  account, e.g.
-  `{github_user_id}+{github_username}@users.noreply.github.com` or
-  `{github_username}@users.noreply.github.com`.
+- [Gitter's export tool](https://github.com/minrk/archive-gitter) does not export
+  workspace settings, so you will need to [configure the settings for your Zulip
+  organization](/help/customize-organization-settings). This includes settings
+  like [email visibility](/help/restrict-visibility-of-email-addresses),
+  [message editing permissions](/help/configure-message-editing-and-deletion#configure-message-editing-and-deletion_1),
+  and [how users can join your organization](/help/restrict-account-creation).
 
-  Since one cannot receive email at those noreply email addresses,
-  imported users will need to use GitHub authentication to log in to
-  Zulip and will be unable to receive email notifications until they
-  [change their Zulip email address](/help/change-your-email-address).
+- Gitter's export tool does not export user settings, so users in your organization
+  may want to [customize their account settings](/help/getting-started-with-zulip).
+
+- The [Gitter API][gitter-api-user-data] doesn't contain data on which users are
+  administrators of a Gitter channel.  As a result, all Gitter users are imported
+  into Zulip as [members](/help/roles-and-permissions).
+
+- Gitter's export tool doesn't export email addresses, only GitHub usernames.
+  Zulip's import tool will set GitHub as the only authentication method enabled
+  by default to avoid user confusion.
```

Comment:
We seem to have lost the instruction to change one's email address after creating an account. That instruction is by far the most important caveat -- The product is in a semi-broken state for users who don't do that. Long term, we probably want to arrange that the first thing that happens when a Gitter user tries to sign in is prompting them to select a different email address associated with their GitHub account to use. @mateuszmandera can you start a #backend thread about this? (It's also possible I've forgotten some clever thing we already do here).

But in the meantime, we should restore the text about how users need to change their email address so we can merge this -- @drrosa can you do that?

Reply:
@timabbott We figured it was best to move the instruction to change one's email address to the first paragraph of the **How users will log in for the first time** section of the page as noted in [this comment](https://github.com/zulip/zulip/pull/23668#issuecomment-1344706108) and shown in [this screenshot](https://github.com/zulip/zulip/pull/23668#issuecomment-1366909489). Does that work for you?

Reply:
Ahh, OK, I missed that. That's reasonable.